tools/ports/__init__.py

```diff
@@ -69,9 +68,13 @@ def get_all_files_under(dirname):
 def dir_is_newer(dir_a, dir_b):
   assert os.path.exists(dir_a)
   assert os.path.exists(dir_b)
-  newest_a = max([os.path.getmtime(x) for x in get_all_files_under(dir_a)])
-  newest_b = max([os.path.getmtime(x) for x in get_all_files_under(dir_b)])
-  return newest_a < newest_b
+  files_a = [(x, os.path.getmtime(x)) for x in get_all_files_under(dir_a)]
+  files_b = [(x, os.path.getmtime(x)) for x in get_all_files_under(dir_b)]
+  newest_a = max([f for f in files_a], key=lambda f: f[1])
+  newest_b = max([f for f in files_b], key=lambda f: f[1])
+  logger.debug('newest_a: %s %s', *newest_a)
+  logger.debug('newest_b: %s %s', *newest_b)
```

Comment:
```suggestion
```

Reply:
These are `.debug` so will only be shown in EMCC_DEBUG=1 mode and they are useful for debugging.  The other logging below is at the `warning` level so it get shown by default.